docs/integrations/databases/open-telemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+id: redis
+title: Sumo Logic Open Telemetry App for Redis
+sidebar_label: Redis
+description: Documentation for Open Telemetry App for Redis.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/databases/redis.png')} alt="Thumbnail icon" width="50"/>
+
+[Redis](https://redis.io/docs/about/) is an in-memory data structure implementing a distributed, in-memory key-value database with optional durability.
+
+The Sumo Logic App for Redis helps you monitor the Redis database cluster. The preconfigured dashboards provide detailed analysis based on logs. The panels provide details such as RDBMemory Usage, events, RDB and AOF events.
+
+Redis logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/schematics-redis.png')} alt="Redis Schematics" />
+
+##Redis Log Types
+
+This Sumo Logic App for Redis supports Logs for Redis in Cluster or Standalone mode. The Redis logs are generated in files as configured in the cluster node configuration files in `/etc/redis/redis.conf`. For more details on Redis logs, [click here](https://redislabs.com/ebook/part-2-core-concepts/chapter-5-using-redis-for-application-support/5-1-logging-to-redis/).
+
+##Fields creation in Sumo Logic for Redis
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/), which will be created as part of Redis App install if not already present.
+
+**db.cluster.name** - User configured. Enter a name to identify this Redis cluster. This cluster name will be shown in the Sumo Logic dashboards.
+**db.system** - Has fixed value of redis
+**deployment.environment** - User configured. This is the deployment environment where the Redis cluster resides. For example: dev, prod or qa.
+**sumo.datasource** - has fixed value of redis
+
+##Prerequisite
+
+This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment for the Sumo Logic App for Redis. By default, Redis logs are stored in a log file.
+
+Follow the instructions to set up log collection:
+
+1. To configure the Redis log file, locate your local `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` configuration file in the database directory. By default, Redis logs are stored in `/var/log/redis/redis-server.log`.
+
+After determining the location of conf file modify the `redis.conf` configuration file logging parameters:
+
+  *   Open `redis.conf` configuration file in a text editor.
+  *   Set the following config parameters:
+    *   Specify the server verbosity level. The value loglevel in conf file can be set to one of the following:
+      *   debug (a lot of information, useful for development/testing)
```

Comment:
I think it's better if we remove this parenthesis, and define the terms, a one-line description is enough.

Reply:
The same should be followed in 47, 48,49, 50